core/src/main/java/org/codice/imaging/nitf/core/tre/impl/TreParser.java

```diff
@@ -79,7 +83,24 @@ public TreParser() throws NitfFormatException {
     }
 
     private void unmarshal(final InputStream inputStream) throws JAXBException {
-        tresStructure = (Tres) getUnmarshaller().unmarshal(inputStream);
+        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+
+        dbf.setExpandEntityReferences(false);
+
+        Document document;
+        try {
+            DocumentBuilder db = dbf.newDocumentBuilder();
+            document = db.parse(inputStream);
+        } catch (SAXException | IOException e) {
+            LOG.warn("Error parsing input. Set log to DEBUG for more information.");
+            LOG.debug("Error parsing input. {}", e);
+            throw new JAXBException(e);
+        } catch (ParserConfigurationException e) {
+            LOG.error("Error creating DocumentBuilder. Set log to DEBUG for more information.");
+            LOG.debug("Error creating DocumentBuilder. {}", e);
+            throw new JAXBException(e);
+        }
+        tresStructure = (Tres) getUnmarshaller().unmarshal(document);
```

Comment:
Maybe that's the solution then, if we're sure this particular unmarshaller will never be susceptible to entity expansion, then we can probably consider this particular issue a "Not an issue"